CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# CHANGELOG
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## Unreleased
+
+### Added
+
+### Fixed
```

Comment:
personally i don't like having these extra sections under unreleased until they have values in them, but it doesn't matter for this pr!